modules/kubernetes/fluxcd-v2/variables.tf

```diff
@@ -0,0 +1,47 @@
+variable "azdo_pat" {
+  description = "PAT to authenticate with Azure DevOps"
+  type        = string
+}
```

Comment:
@phillebaba shouldn't we read this from keyvault by default? Example: https://github.com/XenitAB/terraform-modules/blob/0b076eaab9b2b37073ce0a3eaca5dfc20efd7379/modules/kubernetes/azdo-proxy/variables.tf#L1-L16

Reply:
No i think it is better if it is passed into the module.